x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -150,6 +161,23 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
 
       const { savedObjectsTaggingOss, ...startPlugins } = startPluginsDeps;
 
+      const query = this.queryService.start({
+        uiSettings: core.uiSettings,
+        storage: this.storage,
+        http: core.http,
+      });
+
+      const filterManager = new FilterManager(core.uiSettings);
```

Comment:
To be fair we should probably be doing something similar to initialize a new FilterManager instance for timeline, rather than creating it at the React component level.